# Hekateros hekateros_control/CloseGripper service

---
